apps/calendar/locales/calendar.en-US.properties

```diff
@@ -79,6 +79,10 @@ remove-account-dialog-details=Removing this account will also remove your local
 # for accessibility purposes, not visible to the eye.
 drawer-sync-button=Sync
 
+# sync progress
```

Comment:
Add a comment like this: https://github.com/jobara/gaia/blob/1086930/apps/calendar/locales/calendar.en-US.properties#L254-L255 here